srcalc/src/test/java/gov/va/med/srcalc/domain/model/AbstractVariableTest.java

```diff
@@ -25,6 +27,76 @@
+            + "2345678901234567890123456789012345678901234567890123456789012345"
+            + "6789012345678901234567890123456789012345678901234567890123456789"
+            + "0123456789012345678901234567890123456789012345678901234567890123"
+            + "4567890123456789012345678901234567890123456789012345678901234567"
+            + "8901234567890123456789012345678901234567890123456789012345678901"
+            + "2345678901234567890123456789012345678901234567890123456789012345"
+            + "6789012345678901234567890123456789012345678901234567890123456789"
+            + "0123456789012345678901234567890123456789012345678901234567890123"
+            + "4567890123456789012345678901234567890123456789012345678901234567"
+            + "8901234567890123456789012345678901234567890123456789012345678901"
+            + "2345678901234567890123456789012345678901234567890123456789012345"
+            + "6789012345678901234567890123456789012345678901234567890123456789"
+            + "0123456789012345678901234567890123456789012345678901234567890123"
+            + "4567890123456789012345678901234567890123456789012345678901234567"
+            + "8901234567890123456789012345678901234567890123456789012345678901"
+            + "2345678901234567890123456789012345678901234567890123456789012345"
+            + "6789012345678901234567890123456789012345678901234567890123456789"
+            + "0123456789012345678901234567890123456789012345678901234567890123"
+            + "4567890123456789012345678901234567890123456789012345678901234567"
+            + "8901234567890123456789012345678901234567890123456789012345678901"
+            + "2345678901234567890123456789012345678901234567890123456789012345"
+            + "6789012345678901234567890123456789012345678901234567890123456789"
+            + "0123456789012345678901234567890123456789012345678901234567890123"
+            + "4567890123456789012345678901234567890123456789012345678901234567"
+            + "8901234567890123456789012345678901234567890123456789012345678901"
+            + "2345678901234567890123456789012345678901234567890123456789012345"
+            + "6789012345678901234567890123456789012345678901234567890123456789"
+            + "0123456789012345678901234567890123456789012345678901234567890123"
+            + "4567890123456789012345678901234567890123456789012345678901234567"
+            + "8901234567890123456789012345678901234567890123456789012345678901"
+            + "2345678901234567890123456789012345678901234567890123456789012345"
+            + "6789012345678901234567890123456789012345678901234567890123456789"
+            + "0123456789012345678901234567890123456789012345678901234567890123"
+            + "4567890123456789012345678901234567890123456789012345678901234567"
+            + "89012345678901234567890123456789";
+
+    public static final String FOUR_THOUSAND_ONE_CHARS = FOUR_THOUSAND_CHARS + "X";
```

Comment:
Wouldn't it have been easier to use a static initializer here instead of storing all those? I know there's a trade off and I'm not sure which is better.